tensorflow/lite/python/lite.py

```diff
@@ -668,7 +668,7 @@ def __init__(self):
     # When the value is true, the MLIR quantantizer triggers dynamic range
     # quantization in MLIR instead of the old quantizer. Used only if
     # experimental_new_quantizer is on.
-    self.experimental_new_dynamic_range_quantizer = True
+    self.experimental_new_dynamic_range_quantizer = False
```

Comment:
It seems this is the only change that actually happened in this PR. Can you also add tests and remove all irrelevant changes?

Also, this now breaks with the comment.